src/pages/RepoPage/CoverageTab/subroute/ComponentsMultiSelect/ComponentsMultiSelect.tsx

```diff
@@ -60,7 +60,7 @@ export default function ComponentsMultiSelect() {
   }
 
   return (
-    <>
+    <div className="w-1/6">
```

Comment:
This is the same class as on the [FlagsMultiSelect](https://github.com/codecov/gazebo/blob/433e4e9f84eb8b7f02e94982cdda1a1283d31e37/src/pages/RepoPage/CoverageTab/subroute/FileExplorer/FlagMultiSelect.jsx#L69)